src/utils/validateOptions.js

```diff
@@ -52,6 +52,8 @@ export default function (result, ruleName, ...optionDescriptions) {
 
     // If possible is an object ...
     Object.keys(actual).forEach(optionName => {
+      if (optionName === "warn") { return }
```

Comment:
At the moment I can't use the { "warn": true } in any rule option because it fails with Invalid option name "warn" for rule "indentation"

I guess this line fixes it?